Soccar [Game-Server]/src/nl/soccar/gameserver/model/command/listener/BroadcastCommandListener.java

```diff
@@ -8,10 +8,15 @@
 import nl.soccar.library.enumeration.Privilege;
 
 /**
- * @author Lesley
+ * Listener for broadcasting commands.
+ * 
+ * @author PTS34A
  */
 public final class BroadcastCommandListener extends CommandListener {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)